common/rpcclient.c

```diff
@@ -0,0 +1,192 @@
+#include <common/rpcclient.h>
+
+#include <assert.h>
```

Comment:
These seem very much a part of libplugin, TBH.  Maybe that's because we currently don't supply any other C clients, but I'm not sure what we win by moving this here?

Reply:
Happy to move them into `libplugin`, and move them back out again once we have a user for it.

Reply:
Moved to `libplugin.[ch]`, we can move them again once we have a C API that we want to expose outside of the plugins.